src/NuGet/Microsoft.NETCore.Compilers/Microsoft.NETCore.Compilers.Package.csproj

```diff
@@ -1,46 +0,0 @@
-﻿<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
-<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-
-    <IsPackable>true</IsPackable>
-    <NuspecPackageId>Microsoft.NETCore.Compilers</NuspecPackageId>
```

Comment:
Couple of questions:

- I'm not familiar with arcade and how it works, but I'm seeing the package name referenced in a comment here:

 
 
 
 https://github.com/dotnet/arcade/blob/0b5e6fc8489539f9a9a63bb547729722042c001b/src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props#L54

    Anything need to be updated somewhere to use the other package?

- The package has recent releases on NuGet.org. Should it now be marked deprecated/obsolete?

- Also https://github.com/dotnet/format/blob/68502482e34922e02108f6c1e7f7913bfecd1058/eng/Version.Details.xml#L4-L7

- The solution filter still references the deleted packages I think.

- Also docs/wiki/Troubleshooting-tips.md mentions it

- Also eng/config/PublishData.json

Reply:
> I'm not familiar with arcade and how it works, but I'm seeing the package name referenced in a comment [here](https://github.com/dotnet/arcade/blob/0b5e6fc8489539f9a9a63bb547729722042c001b/src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props#L54).

The comment is out of date. The actual code that controls refers to the supported package.

https://github.com/dotnet/arcade/blob/0b5e6fc8489539f9a9a63bb547729722042c001b/src/Microsoft.DotNet.Arcade.Sdk/tools/Compiler.props

> The package has recent releases on NuGet.org. Should it now be marked deprecated/obsolete?

Yes once this is done I'm going to obsolete them. They've been previously listed as obsolete. 

> The solution filter still references the deleted packages I think.

Sigh, we really need to delete `Compilers.sln` or `Compilers.slf`. Having both is just annoying. 

> Also eng/config/PublishData.json

Can fix that up.

Reply:
> Sigh, we really need to delete `Compilers.sln` or `Compilers.slf`. Having both is just annoying.
> 
> >

There is https://github.com/dotnet/roslyn/pull/47531